### ISSUE TITLE
Новый Checkbox

### ISSUE THREAD

**Issue by QLAWQ** [tretyakovsa/Sonoff_WiFi_switch#100]
Предлагаю такой вот вариант чекбокса.
![image](https://user-images.githubusercontent.com/37859458/47617870-192ff100-dadd-11e8-8dfa-71f05488e5bf.png)

Его код: 
       element.innerHTML += '<div class="toggle-button-cover" '+style_val+'><div class="button-cover"><div align="center">'+renameBlock(jsonResponse, obj.title)+'</div><div class="button r '+obj.design+'"><input type="checkbox" id="'+name_val+'" '+action_val+' class="checkbox" '+checked+'><div class="knobs"></div><div class="layer"></div></div></div></div>';
      
Что скажите?

**Comment by QLAWQ** [comment]
Надпись сверху. При отображении на мобильных не наползает на чекбокс.

**Comment by renat2985** [comment]
Да, действительно если описание в title достаточно длинное некоторые символы попадают под кнопку.

Ваше решение к сожалению тоже не идеально, если мы будем использовать большие кнопки текст будет попадать так же под кнопки:
![2018 10 29-14 37 32](https://user-images.githubusercontent.com/1230815/47650401-d166a380-db88-11e8-8716-871d9f4df19d.png)

Добавил новый css, теперь текст будет отображаться вот так
![2018 10 29-14 38 23](https://user-images.githubusercontent.com/1230815/47650415-da577500-db88-11e8-81d8-e4e0960cd4fb.png)
Это не спасет от длинных предложений но текста без ущерба от дизайна теперь можно писать больше

Если есть еще интересные предложения пишите. Но старайтесь более подробно их описывать.

**Comment by gorgonov** [comment]
Добавил на странице управления красивую кнопку, но она не переключает реле.Что не так?
{
"type": "checkbox",
"title": "Свет в теплице 1",
"state": "{{relay1}}",
"design":"one",
"response":"[[relay1]]",
"action":"relay not 1"
},

а на главной странице не знаю, как такую кнопку добавить.

**Comment by gorgonov** [comment]
В логе пишет:
![image](https://user-images.githubusercontent.com/16058719/48675133-af5cb180-eb87-11e8-84b5-cdef14a18aa3.png)

**Comment by renat2985** [comment]
Вот так попробуйте:
```
 {
"type": "checkbox",
"title": "Свет в теплице 1",
"state": "{{relay1}}",
"design":"one",
"action":"/cmd?command=relay not 1"
},
```

Это скорее всего баг, я посмотрю позже, должно работать так как вы написали.

**Comment by gorgonov** [comment]
Работает, переключает. Я уже такой вариант нашел. Но! Первоначальное состояние кнопки всегда On, затем при щелчке переключает. Но показывает не реальное состояние реле, а противоположное предыдущему.

**Comment by gorgonov** [comment]
> а на главной странице не знаю, как такую кнопку добавить.

>

**Comment by renat2985** [comment]
> Работает, переключает. Я уже такой вариант нашел. Но! Первоначальное состояние кнопки всегда On, затем при щелчке переключает. Но показывает не реальное состояние реле, а противоположное предыдущему.

Попробуйте с "state" поиграться, напишите сами туда 1 и потом 0 посмотрите какое состояние кнопка выводит. 1 это должно быть включено, 0 выключено. Если у вас наоборот, посмотрите на странице настройки PIN-ов, включите или выключите инверсию.
